src/Controls/src/Core/HandlerImpl/Toolbar/Toolbar.Android.cs

```diff
@@ -4,40 +4,29 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using Android.Content;
-using Android.Content.Res;
-using Android.Graphics;
-using Android.Graphics.Drawables;
 using Android.Views;
-using AndroidX.AppCompat.App;
-using AndroidX.AppCompat.Graphics.Drawable;
-using AndroidX.DrawerLayout.Widget;
+using Android.Runtime;
 using Google.Android.Material.AppBar;
 using Microsoft.Maui.Controls.Platform;
-using Microsoft.Maui.Graphics;
 using Microsoft.Maui.Handlers;
 
-namespace Microsoft.Maui.Controls.Handlers
+namespace Microsoft.Maui.Controls
 {
-	public partial class ToolbarHandler : ElementHandler<Toolbar, MaterialToolbar>
+	public partial class Toolbar
```

Comment:
This is just moving the `Controls.ToolbarHandler` code into a remapped `Toolbar.cs`, because I moved the ToolbarHandler itself into Core